### DISCUSSION TITLE
Question: Is there a version of Dynamic Trees that targets MC 1.19.X?

### DISCUSSION THREAD

**Discussion by fritogotlayed** [DynamicTreesTeam/DynamicTrees#715] [Q&A] [answered]
My apologies for asking this question in potentially the wrong place. I am returning to MC after many years away and was looking to create a very bare bones modpack to play with my 5 year old. The intent was to add some quality of life mods so the experience is pretty close to what he has seen in videos while still giving the adults a bit better of an experience.

That said I would love to include Dynamic Trees but I am unsure if there is a version that I can get for MC `1.19.X. I've had a peek through a few of the repos and they appear to be targeting different versions of MC. Ex: DynamicTrees is 1.12 and DynamicTreesPlus is 1.16. 

Is there a version of DynamicTrees that targets 1.19.X minecraft? If not, how would I find which version in the DynamicTrees ecosystem supports closest to the latest version of MC?

**Comment by Harleyoc1** [comment]
Our latest supported MC version is currently 1.18, however it is in alpha as our priority is finishing the 1.16 version first. 

You can find all Dynamic Trees versions on [the files page on CurseForge](https://www.curseforge.com/minecraft/mc-mods/dynamictrees/files/all) which you can filter by MC version using the dropdown initially labelled "All versions". All add-ons rely on this so if you can't find a version on here it is not supported by DT or any add-ons. Also note that not all add-ons are updated to the latest MC versions supported by DT itself.

**Reply by fritogotlayed** [reply]
Huge thank you Harleyoc1! I must of looked right over that version number when browsing CurseForge the other day.

**Reply by exocyt0sis** [reply]
> Our latest supported MC version is currently 1.18, however it is in alpha as our priority is finishing the 1.16 version first.
> 
> You can find all Dynamic Trees versions on [the files page on CurseForge](https://www.curseforge.com/minecraft/mc-mods/dynamictrees/files/all) which you can filter by MC version using the dropdown initially labelled "All versions". All add-ons rely on this so if you can't find a version on here it is not supported by DT or any add-ons. Also note that not all add-ons are updated to the latest MC versions supported by DT itself.

Since you posted your reply in August of last year, may I please ask you for an update? How far away do you estimate the 1.19.x update to be?

**Comment by Harleyoc1** [comment, marked as answer]
There is now a 1.19.2 version in beta. Mangroves are currently being worked on after which we can move it into release. 

1.19.3/4 will not be made, it seems most mods are staying on 1.19.2 because of code changes.
